.github/workflows/lint-php.yml

```diff
@@ -19,19 +25,21 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        php-versions: ["7.4", "8.0", "8.1"]
+        php-versions: [ "7.4", "8.0", "8.1", "8.2" ]
```

Comment:
7.4 should be removed

Reply:
https://github.com/nextcloud/announcementcenter/blob/master/appinfo/info.xml#L38
No, as the master branch will support 25+26